src/root.component.tsx

```diff
@@ -2,12 +2,27 @@ import React from "react";
 import { BrowserRouter, Route } from "react-router-dom";
 import Login from "./login/login.component";
 
-export default function Root(props: RootProps) {
-  return (
-    <BrowserRouter basename="/openmrs/spa">
-      <Route to="login" component={Login} />
-    </BrowserRouter>
-  );
+export default class Root extends React.Component {
```

Comment:
Only class components can implement [Error Boundaries](https://reactjs.org/docs/error-boundaries.html), so I had to switch to a class component.